js/autologout.js

```diff
@@ -17,6 +17,9 @@
       if (context != document) {
         return;
       }
+      if (_autologout_prevent()) {
```

Comment:
Not sure how this will work.

There is no definition for   _autologout_prevent